src/api/request_models/user.py

```diff
@@ -56,6 +56,27 @@ def validate_phone_number(cls, value: str):
     def validate_date_of_birth(cls, value: str):
         return datetime.strptime(value, DATE_FORMAT).date()
 
+    def update_db_model(self, user: User) -> User:
+        user.telegram_id = self.telegram_id
+        user.name = self.name
+        user.surname = self.surname
+        user.date_of_birth = self.date_of_birth
+        user.city = self.city
+        user.phone_number = self.phone_number
+        return user
+
+    def compare_with_db_model(self, user: User) -> bool:
+        telegram_id, name, surname = user.telegram_id, user.name, user.surname
+        date_of_birth, city, phone_number = user.date_of_birth, user.city, user.phone_number
+        return (
+            telegram_id == self.telegram_id
+            and name == self.name
+            and surname == self.surname
+            and date_of_birth == self.date_of_birth
+            and city == self.city
+            and phone_number == self.phone_number
+        )

```

Comment:
Зачем лишние временные переменные? Логичнее расположить сравниваемые значения слева, а то с  чем сравниванием - справа.
```suggestion
        return all(
            (
                self.telegram_id == user.telegram_id,
                self.name == user.name,
                self.surname == user.surname,
                self.date_of_birth == user.date_of_birth,
                self.city == user.city,
                self.phone_number == user.phone_number,
            )
        )
```

Reply:
Я в моделе пробовала сравнивать модель со схемой и у меня сравнивались ячейки и всегда было False, но тут это действительно лишнее